changes made on Top Navigation and car card

Pratishah opened the pull request:
deleted unnecessary style classes and comments ,
 removed duplicate test files ,
 changes on top nav styling

Pratishah committed:
made top navigation bar and card styling with tests

Pratishah committed:
design changed in top navbar

Pratishah committed:
deleted css  and map files

Pratishah committed:
Merge branch 'dev_kartik_FeaturedCars' of https://github.com/DteTRPod3/TRTraining_FinalAssignment into dev_pratik_featured_Section1

Pratishah committed:
added tests for card and featured components

kboodi committed:
Changed color for pagination arrow and timer set to 5 sec.
Added Unit testing

kboodi committed:
resolved merge conflicts

kboodi committed:
merge conflicts

kboodi committed:
removed unnecessary import line

kboodi committed:
Merge branch 'dev' of https://github.com/DteTRPod3/TRTraining_FinalAssignment into dev_kartik_FeaturedCars

kboodi committed:
chaned css for pagination arrows

kboodi committed:
Merge branch 'dev' of https://github.com/DteTRPod3/TRTraining_FinalAssignment into dev_kartik_FeaturedCars

kboodi committed:
Added test cases to nav bar

kboodi committed:
deleted css and map files om serach bar

kboodi committed:
css change to View all added icon from figma

Pratishah committed:
conflicts resolved

Pratishah committed:
completed changes on navbar

Pratishah committed:
added test cases for redirection on xtreme_brand_logo_click

Pratishah committed:
removed font-styles class and deleted extra comments and unused test files

pashedin commented:
this PR looks awfully similar to 
https://github.com/DteTRPod3/TRTraining_FinalAssignment/pull/26

Pratishah commented:
Ticket Link:  https://abhirams.atlassian.net/browse/P3ECWFC-8
                    https://abhirams.atlassian.net/browse/P3ECWFC-10
                     https://abhirams.atlassian.net/browse/P3ECWFC-40

deleted unnecessary classnmes , removed dublicate test files and comments, done styling on top Navbar and  Car-card

pashedin commented on src/components/FeatureCars/FeaturedCars.scss:
```diff
@@ -7,39 +9,59 @@
+  margin-top: 2%;
+}
+
+// .viewallbtn {
+//   a {
+//     float: right;
+//     margin-bottom: 5rem;
+//     display: block;
+//   }
+//   a:hover {
+//     color: $black;
+//   }
+// }
```
You know what to do

pashedin commented on src/components/Navbar/TopNavbar.tsx:
```diff
@@ -2,24 +2,44 @@ import React from "react";
+        <div className="collapse navbar-collapse layout--header--container">
+          <ul className="navbar-nav mr-auto">
+            <li className="nav-item layout--header--list">
+              <Link to="" className="nav-link layout--header--link">
```
please do not leave empty links, if there is nowhere to redirect to then add a `#`
